targets/TARGET_Samsung/TARGET_SIDK_S5JS100/PinNames.h

```diff
@@ -21,155 +21,198 @@
 #define MBED_PINNAMES_H
 
 #include "cmsis.h"
-//#include "gpio.h"
-#include "s5js100_pinconfig.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-typedef enum {
-    PIN_INPUT,
-    PIN_OUTPUT
+typedef enum
+{
+	PIN_INPUT,
+	PIN_OUTPUT
 } PinDirection;
 
 #define PORT_SHIFT  5
+#define STATIC_PINMAP_READY 1
+
+typedef enum
+{
+	P00 = 0,
+	P01,
+	P02,
+	P03,
+	P04,
+	P05,
+	P06,
+	P07,
+	P08,
+	P09,
+	P10,
+	P11,
+	P12,
+	P13,
+	P14,
+	P15,
+	P16,
+	P17,
+	P18,
+	P19,
+	P20,
+	P21,
+	P22,
+	P23,
+	P24,
+	P25,
+	P26,
+	P27,
+	P28,
+	P29,
+	P30,
+	P31,
+	P32,
+	P33,
+	P34,
+	P35,
+	P36,
+	P37,
+	P38,
+	P39,
+	P40,
+	P41,
+	P42,
+	P43,
+	P44,
+	P45,
+	P46,
+	P47,
+	P48,
+	P49,
+	P50,
+	P51,
+	P52,
+	P53,
+	P54,
+	P55,
+	P56,
+	P57,
+	P58,
+	P59,
+	P60,
+	P61,
+	P62,
+	P63,
+	P64,
+	P65,
+	P66,
+	P67,
+	P68,
+	P69,
+	P70,
+	P71,
+	P72,
+	P73,
+	P74,
+	P75,
+	P76,
+
+	/* Actually Dummy */
+	UART4_RXD,
+	UART4_TXD,
+	UART5_RXD,
+	UART5_TXD,
+
+	LED1 = P00,
+	LED2 = P01,
+	LED3 = P02,
+	LED4 = P03,
+
+	USI0_RXD_CLK_SCL = P08,
+	USI0_TXD_MOSI_SDA = P09,
+	USI0_RTSN_MISO_NA = P10,
+	USI0_CTSN_CSN_NA = P11,
+	USI1_RXD_CLK_SCL = P12,
+	USI1_TXD_MOSI_SDA = P13,
+	USI1_RTSN_MISO_NA = P14,
+	USI1_CTSN_CSN_NA = P15,
+
+	UART0_RXD = USI0_RXD_CLK_SCL,
+	UART0_TXD = USI0_TXD_MOSI_SDA,
+	UART0_CTS = USI0_CTSN_CSN_NA,
+	UART0_RTS = USI0_RTSN_MISO_NA,
+	UART1_RXD = USI1_RXD_CLK_SCL,
+	UART1_TXD = USI1_TXD_MOSI_SDA,
+	UART1_CTS = USI1_CTSN_CSN_NA,
+	UART1_RTS = USI1_RTSN_MISO_NA,
+
+	UART2_RXD = P27,
+	UART2_TXD = P28,
+	UART2_CTS = P16,
+	UART2_RTS = P17,
+	UART3_RXD = P29,
+	UART3_TXD = P30,
+	UART3_CTS = P18,
+	UART3_RTS = P19,
+
+	SPI0_RXD = P31,
+	SPI0_TXD = P32,
+	SPI0_CLK = P33,
+	SPI0_CSN = P34,
+
+	I2C0_SCL = P35,
+	I2C0_SDA = P36,
+
+	PWM_TOUT0 = P14,
+	PWM_TOUT1 = P15,
+	PWM_TOUT2 = P22,
+	PWM_TOUT3 = P23,
+
+	ADC_CH0 = P73,
+	ADC_CH1 = P74,
+	ADC_CH2 = P75,
+	ADC_CH3 = P76,
+
+    // Arduino connector namings
```

Comment:
Are all these pins available in the "Adapter board"?
It would be good to add comments next to the pin definition to match with the info shown in the PCB silkscreen (J8 and J10).

Reply:
> Are all these pins available in the "Adapter board"?

Yes, we're actually chip providers, so must provided all pins

> t would be good to add comments next to the pin definition to match with the info shown in the PCB silkscreen (J8 and J10).

PCB silksscreen (J8 and J10) is defined by our own development board, which does not meet the requirements of other module manufacturers using our chips